Lib/test/test_syntax.py

```diff
@@ -1507,6 +1507,27 @@ def test_empty_line_after_linecont(self):
         \
 
 pass
+"""
+        try:
+            compile(s, '<string>', 'exec')
+        except SyntaxError:
+            self.fail("Empty line after a line continuation character is valid.")
```

Comment:
Error message from next test. Something more like "Indented statement over multiple lines is valid."

Reply:
:+1: